model/src/main/kotlin/utils/PackageConfigurationProvider.kt

```diff
@@ -31,4 +31,9 @@ interface PackageConfigurationProvider {
      * Return the first matching [PackageConfiguration] for the given [packageId] and [provenance] if any.
      */
     fun getPackageConfiguration(packageId: Identifier, provenance: Provenance): PackageConfiguration?
+
```

Comment:
Commit message: What do you mean by "To configuration ..."?

Reply:
Updated the message.